static/analysis.go

```diff
@@ -117,6 +127,24 @@ func analyzeHandler(
 					message.CommandRole,
 				)
 			}
+		case "ConsumesEventType":
+			if mi, ok := args[0].(*ssa.MakeInterface); ok {
```

Comment:
Same deal here, there might be an opportunity to reduce a little bit of the repetition here.

Reply:
Not sure if I understand how we can reduce the repetition here.

Reply:
I'm just imagining something a bit like I described above, so perhaps a function like `addMessageFromArguments(args, hdr.MessageNamesValue.Produced, message.CommandRole)`.

Reply:
Ok, that should be addressed in https://github.com/dogmatiq/configkit/pull/130/commits/860b0a6ba50166e5803e5c5918140b53f50f38e2.